docs/getting-started/ory-cloud-oauth2.mdx

```diff
@@ -0,0 +1,213 @@
+---
+id: ory-cloud-oauth2
+title: Perform OAuth2 Authorization Code Grant with Ory Cloud
+sidebar_label: OAuth2 in Ory Cloud
+---
+
+This guide shows you a sample implementation of an Authorization Code Grant OAuth2 flow in Ory Cloud that you can use as a
+starting point for using the Ory Cloud OAuth2 Service (Ory Hydra).
+
+The implementation is based on a simple Node.js application with login and consent screens that allow you to simulate a common
+real-world scenario where an application (client) asks the user for consent to get access to certain data or operations (scopes).
+
+For example, a service for selling goods allows users to add pictures from their Google Photos albums to their listings. To get
+access to the user's photos, the OAuth2 client of the goods selling service asks the user for consent to access data on their
+behalf. The request includes scopes - a list of operations the client wants to perform on behalf of the user. When the user
+agrees, the OAuth2 client gets an authorization code which it exchanges for an `id_token` and `access_token` pair issued by Google
+Photos. These tokens allow the client to access the user's resources from an external service.
+
+By following this guide you will:
+
+- Run a simple login and consent Node.js application on your machine.
```

Comment:
explain what it does: The Web UI to let users grant or reject access to their data